nodecore/api-calls-examples/nodecore-pending-transactions-webpage/src/api/utils.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable  @typescript-eslint/no-non-null-assertion */
+
+import { IJsonRPC } from '../models/IJsonRPC';
+
+export const request = async <T>(method: string, params = {}): Promise<T> => {
+    const myHeaders = new Headers();
+    myHeaders.append('Content-Type', 'application/json');
+
+    const raw = JSON.stringify({
+        jsonrpc: '2.0',
+        method: method,
+        params: params,
+        id: 1
+    });
+
+    const requestOptions: RequestInit = {
+        method: 'POST',
+        headers: myHeaders,
+        body: raw,
+        redirect: 'follow'
+    };
+
+    const response = await window.fetch('/api', requestOptions);
```

Comment:
This page is faced to a client. So client is expected to expose /api endpoint with a nodecore?

Reply:
This was a request from @timstall. @PCasafont already told me that. Maybe we should create kind of a proxy server. A very small node instance using express, for example, where we add the endpoints that we need to expose to the Frontend and We read from there. 

Although, I've implemented some checksum-like validation so we guarantee that, for now, only `getPendingTransactions` method is called. 

Anyway, If You run the project You will see that the requests don't have the NodeCore address but the Frontend address which looks like requests are being made at the same level as the Frontend.